### DISCUSSION TITLE
Question: generate CHANGELOG.md for existing git repo with many tags

### DISCUSSION THREAD

**Discussion by bethmal** [semantic-release/semantic-release#1983] [Q&A]
Hi,

We have an existing gitlab repo. which contains many tags created previously. How can I configure semantic-release/changelog to create CHANGLOG.md for all of those previously created tags, not only for the latest version bumped by semantic-release?

Thanks.

**Comment by trallnag** [comment]
Not possible according to https://github.com/semantic-release/semantic-release/discussions/2368#discussioncomment-2190191.

You could take a look at `conventional-changelog` which is (I think) used under the hood to generate the notes. See this issue for more: https://github.com/conventional-changelog/conventional-changelog/issues/927
